.github/workflows/team_label.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened]
+    branches:
+      - 'main'
+name:
+  team-label
+jobs:
+  team-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: JulienKode/team-labeler-action@v0.1.1
```

Comment:
does this work for labeling issues as well?

Reply:
No, it's an open request and a bit of a gap.